app/commands/payments/stripe/handle_event.rb

```diff
@@ -0,0 +1,22 @@
+class Payments::Stripe::HandleEvent
```

Comment:
I've extracted this bit of logic to be re-usable

Reply:
Nice.